libsolidity/ast/ASTJsonConverter.cpp

```diff
@@ -704,13 +704,13 @@ bool ASTJsonConverter::visit(ElementaryTypeNameExpression const& _node)
 
 bool ASTJsonConverter::visit(Literal const& _node)
 {
-	char const* tokenString = Token::toString(_node.token());
+	string tokenString = literalTokenKind(_node.token());
```

Comment:
Seems that `tokenString` is not used anywhere, but in that single place, maybe it makes sense to only keep the call there.